### DISCUSSION TITLE
ubuntu远程服务器，一直认证失败怎么办？

### DISCUSSION THREAD

**Discussion by Young-Zhen** [houtianze/bypy#651] [Q&A]
在ubuntu远程服务器上pip intsall bypy之后， bypy info之后一直认证超时，请问有什么解决办法吗？

**Comment by houtianze** [comment]
`bypy -dv` 贴一下log。

**Reply by Young-Zhen** [reply]
哎，最后发现好像是因为服务器没有联网QAQ，现在显示
<img width="277" alt="image" src="https://github.com/houtianze/bypy/assets/57033200/f0d052ea-c474-482d-ab90-6a363e678577">
请问是不是已经认证成功了？

**Reply by Young-Zhen** [reply]
已经能够开始下载了，感谢！
